### DISCUSSION TITLE
2023-01-02-low-code

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#131] [General]
# 什么是比较好的低代码产品 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Technology说到低代码，喜欢的人特别喜欢，不喜欢的人很不喜欢，此外也有“假装”去喜欢的，也有喜欢得不明不白的，我现在对于低代码是有点儿喜欢的那种，不过只限于在特定领域，实现需求的速度比熟练工程师写代码要快的场景，最近在思考团队下一步的提效方式，来看看业界开源的低代码产品，说不定会有一些输入。

https://tw93.fun/2023-01-02/low-code.html

**Comment by turkyden** [comment]
太强了！低代码已经进化到这么酷了，最近这几年发展得好快，某位大佬说过，所有管理端系统，归根结底就是 excel，哈哈，excel 是无代码的鼻祖。

**Reply by tw93** [reply]
哈哈哈 赞同 而且非技术用得很溜

**Comment by Yunkou** [comment]
我最近在做流程编排低代码。 魔改过n8n, 但是放弃了，不太适合开源改商用。后来自己写了。

本来是走正规产品序列的但是发现Sass 的售前用它快速打单，和在系统集成过程中砍掉外包团队（大幅度缩减预算）表现异常两眼

**Comment by alswl** [comment]
「通过少写代码，用更便捷的方式来实现原本需写代码的工作」，好观点。这样讲起来，RoR 的 ActiveRecord + 配套的 Admin 页面都可以被认为是低代码的一种实现。

有些论点认为低代码是不需要学习就能使用的代码生成器，而真实的低代码还是需要对数据的组织形态、流转有深刻理解才能掌控好。

**Reply by tw93** [reply]
可以理解成对一类场景的前后端开发模式高度抽象，然后上层通过易用的操作来完成 “搭建组织”
